modules/ROOT/pages/configure-teams.adoc

```diff
@@ -62,6 +62,19 @@ The permissions you selected appear alongside the business groups and environmen
 . Select *Apply permissions*. +
 The permissions you selected now apply to team members and appear in the *Permissions* section for your team.
 
+=== View Team Permissions Limits
+
+Each team, including your root organization's team, has a *Limits* section that shows how close it is to hitting limits imposed by Anypoint Platform.
+
+To view limits:
+
+include::partial$include-nav-teams.adoc[]
+
+. Click the team for which you want to view limits.
+. Click the *Limits* tab.
+
```

Comment:
Let's add: 
> Please Note: The number of permissions used by a Team may reflect internal permissions granted automatically by the system and which are not necessarily visible or manageable directly in the UI.

Reply:
I think saying "...number of permissions in use by a team..." or "...number of permissions used by a team..." makes _slightly_ more sense 
1. because the term "granted" is only in use once in the sentence and
2. (more importantly), in the UI, it refers to them being "Used"
<img width="769" alt="Screen Shot 2023-01-13 at 2 57 06 PM" src="https://user-images.githubusercontent.com/1872517/212417740-e0e1f90a-8281-4998-b203-0097f9d2e3d6.png">